distributed/scheduler.py

```diff
@@ -282,78 +283,188 @@ class WorkerState:
 
     # XXX need a state field to signal active/removed?
 
+    _actors: set
+    _address: object
+    _bandwidth: int
```

Comment:
Sometimes we use int and sometimes Py_ssize_t, can you help explain why we choose one over the other?  From a readability perspective I would prefer int everywhere.  I'm guessing that this varies a bit on 32bit systems?

Reply:
Moved to [this thread]( https://github.com/dask/distributed/pull/4294/files#r535543341 )